src/specific_endian.rs

```diff
@@ -456,11 +452,11 @@ mod tests {
         }
         let foo: BigEndian<EndianAwareExample> = EndianAwareExample::LittleEndianFunction(0xf0).into();
         #[allow(unused_assignments)]
-        let mut value = 0;
-        match foo.to_native() {
-            EndianAwareExample::BigEndianFunction(v) => { println!("be: {:x}", v); value = v }
-            EndianAwareExample::LittleEndianFunction(v) => { println!("le: {:x}", v); value = 0 }
-        }
+        let value = match foo.to_native() {
+            EndianAwareExample::BigEndianFunction(v) => v,
+			// TODO this doesn't seem right? It'll cause the assert to always fail.
+            EndianAwareExample::LittleEndianFunction(_v) => 0, 
+        };
         assert_eq!(value, 0x0f000000000000000);
```

Comment:
This doesn't seem right to me? If `to_native` would return `LittleEndianFunction` on a different platform the test would fail because `0 != 0x0f000000000000000`